### PR TITLE
Update project: Enhance quickstart

### PR DESCRIPTION
**Pull request by satwikkansal**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack e7efe4af475f0124b71ef78ca0c474a5aa7f094f

File: data/projects/enhance_coala_quickstart.md [hunk 1]
```diff
@@ -13,43 +13,51 @@ However, there’s scope for improvement in the following areas:
...
+ [BONDING] https://gitlab.com/coala/GSoC-2017/milestones/6
+ [PHASE 1] https://gitlab.com/coala/GSoC-2017/milestones/20
+ [PHASE 2] https://gitlab.com/coala/GSoC-2017/milestones/21
+ [PHASE 3] https://gitlab.com/coala/GSoC-2017/milestones/22
+ [STRETCH GOALS] https://gitlab.com/coala/GSoC-2017/milestones/41
+
+##### [PREPARATION/BONDING](https://gitlab.com/coala/GSoC-2017/milestones/6)
 
-* The applicant has identified all the kinds of configuration files to be worked
+* Identified all the kinds of configuration files to be worked
```
Comment (Adrianzatreanu) [inline data/projects/enhance_coala_quickstart.md]: wouldnt it be a good idea to somehow mark those in progress and those done? Like "*" before would mean its done, and it should be mentioned at the bottom of the page.

Comment (satwikkansal) [inline data/projects/enhance_coala_quickstart.md]: I thought of doing so but other projects are not doing it and the "*"s will need to be maintained continuously. It would be easier to maintain if we can do it for phases instead of tasks inside the phases.

Comment (satwikkansal) [inline data/projects/enhance_coala_quickstart.md]: Okay, as of now I've added dates for the milestones and created https://github.com/coala/projects/issues/301. Maybe this should suffice for now :sweat_smile:

**Comment by Adrianzatreanu** [comment]
reack f7a35076c52f0283756f6cf1d0e4752b4c93cf9b

File: data/projects/enhance_coala_quickstart.md (hunk 1 above)
Comment (Adrianzatreanu) [inline data/projects/enhance_coala_quickstart.md]: well, whats a better way than starting it rather than doing it yourself :D ? @Nosferatul is doing it as well, you might just mention at the end: "Note: Those marked with * are ... " and then it would be an improvement brought by you, not something required by all projects.
you can do that in another commit if you wish, i dont mind.

**Comment by Adrianzatreanu** [comment]
ack 2d082c881fd737200c5bfa4337d337fd54b8e66e

**Comment by Adrianzatreanu** [comment]
@rultor merge

File: data/projects/enhance_coala_quickstart.md (hunk 1 above)
Comment (satwikkansal) [inline data/projects/enhance_coala_quickstart.md]: yeah, you're right, I'll add another commit in a while marking the completed tasks with "*".
